from reference_data import global_vars as glova
import navigators as nav

from selenium.common.exceptions import TimeoutException


def collect_team_data():
    # try:
    #     nav.check_connected()
    #     driver = nav.launch_webbrowser(launch_info, nav_followup=True)
    #     df_main_team_data = collect_my_team_data(driver)
    # except TimeoutException:
    #     driver.quit()
    # except nav.NoInternetException:
    #     sys.exit(0, "Reconnect to Wifi before launching script again.")
    return

